.php_cs.dist

```diff
@@ -1,5 +1,14 @@
 <?php
 
+$header = <<<TXT
+Copyright (c) TBD
```

Comment:
@chartjes 

We can easily specify the year here using `date('Y')`, if you like!

Reply:
Might just be a bit of an issue when people contribute after midnight in Europe, hehe, when you guys are still preparing to celebrate a new year.

🎉

Reply:
@localheinz If we add in `date('Y')` then I am okay with this